src/policy/fees.cpp

```diff
@@ -338,8 +325,8 @@ double TxConfirmStats::EstimateMedianVal(int confTarget, double sufficientTxVal,
         failBucket.leftMempool = failNum;
     }
 
-    LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d %s%.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
-             confTarget, requireGreater ? ">" : "<", 100.0 * successBreakPoint, decay,
+    LogPrint(BCLog::ESTIMATEFEE, "FeeEst: %d > %.0f%% decay %.5f: feerate: %g from (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out) Fail: (%g - %g) %.2f%% %.1f/(%.1f %d mem %.1f out)\n",
+             confTarget, 100.0 * successBreakPoint, decay,
              median, passBucket.start, passBucket.end,
              100 * passBucket.withinTarget / (passBucket.totalConfirmed + passBucket.inMempool + passBucket.leftMempool),
```

Comment:
While not introduced in this PR we have an existing floating-point division by zero here that would be really nice to get rid of while you're touching these lines :)

You can see it by running …

```
$ ./configure --with-sanitizers=undefined
$ make
$ src/test/test_bitcoin
```

... and look for `policy/fees.cpp:…:…: runtime error: division by zero` in the output.

When this has been addressed the following UBSan suppression can be removed too:

https://github.com/bitcoin/bitcoin/blob/30dd562fd2c58536fa026fff9853b8d825216b01/test/sanitizer_suppressions/ubsan#L3

Reply:
Nice catch ! Will push a new commit, thanks :)

Reply:
Added in 24bcbbfb7b21641b6f2d00ae18d69c59a7c1f56e